indexer/bin/indexer.js

```diff
@@ -6,6 +6,12 @@ export default {
    * @returns {Promise<Response>}
    */
   async fetch(request, env, ctx) {
+    const secretHeaderKey = env.SECRET_HEADER_KEY
```

Comment:
Why are we treating the secret header key (header name) as a sensitive secret? IMO, we can hard-code this value inside `indexer.js` in plain text.

I mean, like, most web services use the well-specified `Authorization` header name, there is no secrecy around that name.

Reply:
This is the name in goldsky: "Secret header key". I don't care too much about whether it's a secret or not. This is already implemented.